src/store/registration/saga.ts

```diff
@@ -140,4 +143,21 @@ export function* saga() {
     const action = yield take(SagaActionTypes.UpdateProfile);
     success = yield call(updateProfile, action);
   } while (!success);
+
+  yield spawn(conversationsWatcher);
+}
+
+export function* channelsLoaded() {
+  const existingConversationsList = yield select(rawConversationsList());
+  if (existingConversationsList.length > 0) {
+    yield put(setActiveMessengerId(existingConversationsList[0]));
+  }
+}
+
+function* conversationsWatcher() {
+  const channel = yield call(conversationsChannel);
```

Comment:
i don't follow this part. This is calling a saga function `conversationsChannel` which is calling a "multicast" channel. Is it fetching/loading our channel(s) info?

Reply:
In this line we're just getting access to the channel. The next line "listens" for events on that channel with the `take` call in the same way all the other `take` calls work. When an event comes it'll move on.